src/index.ts

```diff
@@ -100,9 +110,12 @@ export function throttling(octokit: Octokit, octokitOptions: OctokitOptions) {
     octokit.log.warn("Error in throttling-plugin limit handler", e),
   );
 
-  // @ts-expect-error
   state.retryLimiter.on("failed", async function (error, info) {
-    const [state, request, options] = info.args;
+    const [state, request, options] = info.args as [
+      State,
+      OctokitResponse<any>,
```

Comment:
Would it be less intrusive to use this:

```suggestion
      OctokitResponse<any> & { retryCount: number },
```

Instead of re-declaring module `@octokit/types`? Or is there an issue you were facing I'm not seeing?

Reply:
Since the module adds a property to the request, we need to augment the types with that property so it's available to end users when they use this plugin.